host-controller/src/main/java/org/jboss/as/domain/controller/logging/DomainControllerLogger.java

```diff
@@ -741,6 +741,11 @@
     @Message(id = 77, value = "Duplicate included socket binding group '%s'")
     XMLStreamException duplicateSocketBindingGroupInclude(String s);
 
-    @Message(id = 78, value = "The profile clone operation is not available on the host '%s'")
+    @Message(id = 78, value = "The profile clone operation is not available on the host '%s'. To be able to use it in a " +
+            "mixed domain, you need to either: " +
+            "a) Make sure that all older slaves with a model version smaller than 4.0.0 " +
+            "ignore the cloned profile and the profile specified in the 'to-profile' parameter. " +
```

Comment:
Do we need to require that they ignore the cloned profile? Since we are rejecting, I don't think it matters. The clone op doesn't alter the cloned profile.